### ISSUE TITLE
Skin necrosis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6078]
**Preferred term label:**
Skin necrosis

**Synonyms**
Cutaneous necrosis

**Definition (free text, please give PubMed ID)**
The presence of cell death (necrosis) affecting the skin

**Parent term (use hpo.jax.org/app)**
Dermatological manifestations of systemic disorders

**Comment by pnrobinson** [comment]
added as HP:0033126 (Label Cutaneous necrosis)
